### DISCUSSION TITLE
Alternative to docker cp.

### DISCUSSION THREAD

**Discussion by rodrigorootrj2** [containerd/nerdctl#786] [Q&A] [answered]
Hi all, i have had use docker cp for send and get files in run container . What is alternative for that?

**Comment by AkihiroSuda** [comment]
The container file system is visible as /proc/PID/root from the host, and you can just use `cp` command.

**Reply by rodrigorootrj2** [reply]
Hi AkihiroSuda, good morning.
I saw on readme, the cp is a "Unimplemented Docker command".
Do you have a example about cp command with nerdctl?

**Reply by fahedouch** [reply]
@rodrigorootrj2  under construction https://github.com/containerd/nerdctl/pull/643 :)

**Comment by AkihiroSuda** [comment, marked as answer]
`nerdctl cp` is now available in nerdctl v0.19: https://github.com/containerd/nerdctl/pull/995
